app/models/user.py

```diff
@@ -352,3 +355,19 @@ def __setattr__(self, name, value):
 @event.listens_for(User, 'init')
 def receive_init(target, args, kwargs):
     target.signup_at = datetime.now(pytz.utc)
+
+
+@event.listens_for(User, 'after_update')
```

Comment:
@poush any particular reason why you think email service should be implemented at the model level instead of the API resource level. In fact I think we should keep models as light as possible and not implement services over there. @niranjan94 you views?